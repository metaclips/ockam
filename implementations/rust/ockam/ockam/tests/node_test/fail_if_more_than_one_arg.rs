#[ockam::test]
async fn my_test(mut c: ockam_node::Context, _x: u64) -> ockam_core::Result<()> {
    c.shutdown_node().await.unwrap();
}

fn main() {}
