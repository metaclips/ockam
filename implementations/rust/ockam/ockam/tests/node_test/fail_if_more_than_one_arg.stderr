error: the function must have exactly one argument
 --> tests/node_test/fail_if_more_than_one_arg.rs:2:18
  |
2 | async fn my_test(mut c: ockam_node::Context, _x: u64) -> ockam_core::Result<()> {
  |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: the `Context` argument must be passed as reference
 --> tests/node_test/fail_if_more_than_one_arg.rs:2:18
  |
2 | async fn my_test(mut c: ockam_node::Context, _x: u64) -> ockam_core::Result<()> {
  |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
